example/web/src/layout.html

```diff
@@ -28,6 +28,7 @@
         </div>
       </div>
     </div>
+    <script src="https://www.recaptcha.net/recaptcha/api.js"></script>
```

Comment:
is this the URL that can be accessed from china?

Reply:
Yeah